pb_plugins/dcsdkgen/utils.py

```diff
@@ -34,10 +34,9 @@ def is_response(struct):
 
 def is_struct(struct):
     """ Check if the message is a data structure or an rpc
-    request/response/result """
+    request/response"""
     return (not struct.name.endswith("Request") and
-            not struct.name.endswith("Response") and
-            not struct.name.endswith("Result"))
+            not struct.name.endswith("Response"))
```

Comment:
Can't e.g. an ActionResult be a struct? I am thinking of this one: https://github.com/Dronecode/DronecodeSDK-Proto/blob/74b6477739fac183b14fbc8df64fc9f71a538c06/protos/action/action.proto#L92

Reply:
`ActionResult` is a struct, but not `*Request` and `*Response`. I was ignoring `Result`s before, but I think I was wrong.

Reply:
Actually I think I just missed the `not`. Should be fine!